Documentation/Assignment 2/tex/assign_2_functional.tex

```diff
@@ -0,0 +1,46 @@
+\chapter{Functional Requirements}
+
+\section{Introduction}
+In this document the requirements for assignment 2 will be stated. There will be improvements to the code, as well as new functionalities. \noindent This will be done using the \fnurl{MoSCoW Model}{https://en.wikipedia.org/wiki/MoSCoW_method}.
+
+\section{Must Have}
+\begin{itemize}
+\itemsep0em
+\item The player must be able to fall through the floor.
```

Comment:
You don't want the player to fall through the floor, you want it to fall through a hole in the floor.